ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Ledger/NetworkProtocolVersion.hs

```diff
@@ -36,10 +36,7 @@ instance SupportedNetworkProtocolVersion ByronBlock where
         -- combinator, not supported by Byron-only.
       ]
   supportedNodeToClientVersions _ = Map.fromList [
-        (NodeToClientV_1, ByronNodeToClientVersion1)
-        -- Enable the LocalStateQuery protocol, no serialisation changes
-      , (NodeToClientV_2, ByronNodeToClientVersion1)
-        -- V_3 enables the hard fork, not supported by Byron-only.
+        (NodeToClientV_9, ByronNodeToClientVersion1)
```

Comment:
Should we just support all versions? Mapping them all to `ByronNodeToClientVersion1`?

Reply:
Yes, I think its better to do so.